pandas/tests/series/methods/test_sum.py

```diff
@@ -0,0 +1,13 @@
+import numpy as np
+
+from pandas import Series
+import pandas._testing as tm
+
+
+class TestSeriesSum:
+    def test_sum_uint64(self):
+        # GH 53401
+        s = Series([10000000000000000000], dtype="uint64")
+        result = s.sum()
+        expected = np.uint64(10000000000000000000)
+        tm.assert_almost_equal(result, expected)
```

Comment:
Can you move the test over to `pandas.tests.reductions.test_reductions.TestSeriesReductions`.

Reply:
yes! That seems right place.